build: update roachtest scripts to use --suite instead of tags

RaduBerinde opened the pull request:
There is one more use (`tag:release_qualification`) inside the Release
Qualification script in TeamCity.

Note that we will not remove tag support until the new flags are
backported on all relevant branches, and even after that we will
provide some basic backward compatibility (e.g. convert a single tag
to a suite).

Epic: none
Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/111947)
<!-- Reviewable:end -->

RaduBerinde commented:
Sample build: https://teamcity.cockroachdb.com/viewLog.html?buildId=12073006&buildTypeId=Cockroach_Nightlies_RoachtestNightlyGceBazel&tab=buildResultsDiv&branch_Cockroach_Nightlies=111251

herkolategan reviewed (approved):
> Reviewed 10 of 10 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/111947)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @renatolabs and @srosenberg)



<!-- Sent from Reviewable.io -->

RaduBerinde committed:
build: update roachtest scripts to use --suite instead of tags

There is one more use (`tag:release_qualification`) inside the Release
Qualification script in TeamCity.

Note that we will not remove tag support until the new flags are
backported on all relevant branches, and even after that we will
provide some basic backward compatibility (e.g. convert a single tag
to a suite).

Epic: none
Release note: None

RaduBerinde commented:
bors r+
